Detectors/TPC/base/include/TPCBase/ParameterGEM.h

```diff
@@ -0,0 +1,131 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file ParameterGEM.h
+/// \brief Definition of the parameter class for the GEM stack
+/// \author Andi Mathis, TU München, andreas.mathis@ph.tum.de
+
+#ifndef ALICEO2_TPC_ParameterGEM_H_
+#define ALICEO2_TPC_ParameterGEM_H_
+
+#include <array>
+
+namespace o2 {
+namespace TPC {
+
+/// \class ParameterGEM
+
+class ParameterGEM{
+  public:
+    static ParameterGEM& defaultInstance() {
+      static ParameterGEM param;
+      param.setDefaultValues();
+      return param;
+    }
+
+    /// Constructor
+    ParameterGEM();
+
+    /// Destructor
+    ~ParameterGEM() = default;
+
+    /// Set the default values
+    void setDefaultValues();
+
+    /// Set absolute gain for the stack
+    /// \param absGain1 Absolute gain in GEM 1
+    /// \param absGain2 Absolute gain in GEM 2
+    /// \param absGain3 Absolute gain in GEM 3
+    /// \param absGain4 Absolute gain in GEM 4
+    void setAbsoluteGain(float absGain1, float absGain2, float absGain3, float absGain4);
+
+    /// Set absolute gain for a single GEM in the stack
+    /// \param absGain Absolute gain
+    /// \param GEM GEM of interest
+    void setAbsoluteGain(float absGain, int GEM) { mAbsoluteGain[GEM-1] = absGain; }
```

Comment:
I guess since `GEM` is a local variable it should be lower case `gem`. This is also true for all occurences below.
I would add a comment in the parameter description that `gem` should be 1-4